react/FileSelector/index.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const FileSelector = ({
+  className,
+  disabled,
+  hidden,
+  multiple,
+  processSelected,
+  renderContent,
+  ...inputProps
+}) => (
+  <label role="button" disabled={disabled} className={className}>
+    {renderContent()}
+    <input
+      hidden={hidden}
```

Comment:
Not sure about the `hidden` attribute.
I'm still looking for answers but I'm wondering if the input shouldn't be readable by a screen-reader in this case, and by using the `hidden` attribute it won't.

Reply:
:+1:
It may explain why it was hidden by css on drive (see https://mattermost.cozycloud.cc/cozycloud/pl/97h8xkbeif8t3krcrj4acszify ). So I'm waiting for your feedback on this